localization/english/modifiers/activity_feast_modifiers_l_english.yml

```diff
@@ -0,0 +1,8 @@
+﻿l_english:
+ feast_cavalry_insight_modifier:0 "Cavalry and Aerial Insight"
+ feast_cavalry_insight_modifier_desc:1 "Plotting out theoretical strategies has given this character ideas for how to more efficiently use their cavalry and aerial."
+ feast_heavy_infantry_insight_modifier:0 "Heavy Infantry and Giants Insight"
+ feast_heavy_infantry_insight_modifier_desc:1 "Plotting out theoretical strategies has given this character ideas for how to more efficiently use their giants and heavy infantry."
+ feast_archers_insight_modifier:0 "Archers and Casters Insight"
+ feast_archers_insight_modifier_desc:1 "Plotting out theoretical strategies has given this character ideas for how to more efficiently use their archers and casters."
```

Comment:
Guess what. You overwrote vanilla localization by this file thus 84 lines of localization just disappeared.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104961193-43886480-59ef-11eb-84c2-36c91518ed87.png)
![image](https://user-images.githubusercontent.com/46384992/104961223-500cbd00-59ef-11eb-94d1-569c42aaf1ed.png)

</details>